App/app.rb

```diff
@@ -0,0 +1,163 @@
+require_relative '../student'
+require_relative '../teacher'
+require_relative '../book'
+require_relative '../rental'
+
+class App
+  def run
+    @option = 0
+    @books = []
+    @people = []
+    @rentals = []
+    show_menu
+  end
+
+  def welcome_message
+    puts 'Welcome to School Library App'
+    puts ''
+  end
+
+  def show_options
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def create_student
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    permission = gets.chomp
+    student = Student.new(age, 0, name, parent_permission: %w[Y y].include?(permission))
+    @people << student
+    puts 'Person creaed successfully'
+    puts ''
+  end
+
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp.to_i
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    teacher = Teacher.new(age, specialization, name)
```

Comment:
- [ ] Please address this as suggested

Reply:
Can you kindly check it again? It is working on my side.
![Screenshot from 2022-05-21 20-13-04](https://user-images.githubusercontent.com/35307862/169657790-5e9e317b-ed52-4d84-8259-530cb7d5cd71.png)

Reply:
- [ ] Please check what is wrong on my side I have the same issue
![t](https://user-images.githubusercontent.com/44745855/169658811-2e0c85bd-307c-4673-8827-bbbf0eb40776.png)

Reply:
I have pushed all the changes and have counter-checked and run again on my side. Maybe the project you cloned is not the recent one. Can you kindly see into this?